jxl/src/frame/modular.rs

```diff
@@ -285,6 +309,42 @@ impl FullModularImage {
             &header,
             global_tree,
             br,
-        )
+        )?;
+
+        let maybe_output = |bi: &mut ModularBufferInfo, grid: usize| -> Result<()> {
+            if bi.is_output {
+                let (gw, _) = bi.grid_shape;
+                let g = (grid % gw, grid / gw);
+                on_output(
+                    bi.channel_id,
+                    g,
+                    bi.buffer_grid[grid].data.as_ref().unwrap(),
+                )?;
+                bi.buffer_grid[grid].mark_used();
+            }
+            Ok(())
+        };
+
+        let mut new_ready_transform_chunks = vec![];
```

Comment:
is `vec![]` more idiomatic than `Vec::new()` (just a learning question for me)

Reply:
I believe so...